ui/revenuecatui/src/main/kotlin/com/revenuecat/purchases/ui/revenuecatui/components/ktx/Alignment.kt

```diff
@@ -73,3 +73,31 @@ internal fun TwoDimensionalAlignment.toHorizontalAlignmentOrNull(): Alignment.Ho
         TwoDimensionalAlignment.BOTTOM_LEADING -> Alignment.Start
         TwoDimensionalAlignment.BOTTOM_TRAILING -> Alignment.End
     }
+
+@JvmSynthetic
+internal fun TwoDimensionalAlignment.toVerticalAlignment(): Alignment.Vertical =
```

Comment:
This is very similar to `toVerticalAlignmentOrNull` but I needed one that assumed a vertical alignment for the "center" types. Same for the horizontal version.